.github/workflows/post-merge.yaml

```diff
@@ -45,6 +45,45 @@ jobs:
             echo "No generated changes to push!"
           fi
 
+  generate-cli-docs:
+    name: Sync Generated CLI Docs
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GH_PUSH_TOKEN }}
```

Comment:
If we ever clean up this and its sibling above: this isn't needed to checkout the code.

Reply:
Done in https://github.com/open-policy-agent/opa/pull/4248